src/io/io.cpp

```diff
@@ -79,6 +79,11 @@ void WriteData(Grid3D &G, struct parameters P, int nfile)
   cudaMemcpy(G.C.density, G.C.device, G.H.n_fields*G.H.n_cells*sizeof(Real), cudaMemcpyDeviceToHost);
 
   chprintf( "\nSaving Snapshot: %d \n", nfile );
+  
+  #ifdef HDF5
```

Comment:
This duplicates the addition of H5Open and H5Close currently in dev, I recommend un-duplicating it